rpm.el8.spec

```diff
@@ -0,0 +1,39 @@
+#
+# spec file for package cloud-init-vmware-guestinfo
+#
+
+#################################################################################
+# common
+#################################################################################
+Name:           cloud-init-vmware-guestinfo
+Version:        1.1.0
```

Comment:
Please spec bump this to `1.2.0` since that is the current tag of the repo.

Reply:
I didn't change the version as your commit had that. I included that in the PR so it should be fixed